core/benches/banking_stage.rs

```diff
@@ -287,14 +332,75 @@ fn bench_banking(bencher: &mut Bencher, tx_type: TransactionType) {
     let _unused = Blockstore::destroy(&ledger_path);
 }
 
+fn bench_banking_scaled(bencher: &mut Bencher, packets_per_batch: usize, tx_type: TransactionType) {
+    // ensure the total number of packets does not change when packets_per_batch
+    // is adjusted
+    bench_banking(
+        bencher,
+        packets_per_batch,
+        4 * 192 / packets_per_batch,
+        tx_type,
+    )
+}
+
 #[bench]
-fn bench_banking_stage_multi_accounts(bencher: &mut Bencher) {
-    bench_banking(bencher, TransactionType::Accounts);
+fn bench_banking_stage_batch_size_192_multi_accounts_no_contention(bencher: &mut Bencher) {
+    bench_banking_scaled(bencher, 192, TransactionType::AccountsNoContention);
 }
 
 #[bench]
-fn bench_banking_stage_multi_programs(bencher: &mut Bencher) {
-    bench_banking(bencher, TransactionType::Programs);
+fn bench_banking_stage_batch_size_192_multi_accounts_contention_full(bencher: &mut Bencher) {
+    bench_banking_scaled(bencher, 192, TransactionType::AccountsContentionFull);
+}
+
+#[bench]
```

Comment:
Love to have these tests. May need to adjust CI if it's timed out due to additional benching